applications/RANSApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -31,14 +32,29 @@ void AddCustomUtilitiesToPython(pybind11::module& m)
 {
     namespace py = pybind11;
 
+    using RansScalarVariableDifferenceNormCalculationUtilityType = RansVariableDifferenceNormsCalculationUtility<double>;
+    py::class_<RansScalarVariableDifferenceNormCalculationUtilityType, RansScalarVariableDifferenceNormCalculationUtilityType::Pointer>(m, "ScalarVariableDifferenceNormCalculationUtility")
+        .def(py::init<const ModelPart&, const Variable<double>&>())
+        .def("InitializeCalculation", &RansScalarVariableDifferenceNormCalculationUtilityType::InitializeCalculation)
+        .def("CalculateDifferenceNorm", &RansScalarVariableDifferenceNormCalculationUtilityType::CalculateDifferenceNorm);
+
     m.def_submodule("RansVariableUtilities")
         .def("ClipScalarVariable", &RansVariableUtilities::ClipScalarVariable)
         .def("GetMinimumScalarValue", &RansVariableUtilities::GetMinimumScalarValue)
         .def("GetMaximumScalarValue", &RansVariableUtilities::GetMaximumScalarValue)
-        .def("CopyNodalSolutionStepVariablesList", &RansVariableUtilities::CopyNodalSolutionStepVariablesList);
+        .def("AddAnalysisStep", &RansVariableUtilities::AddAnalysisStep)
+        .def("IsAnalysisStepCompleted", &RansVariableUtilities::IsAnalysisStepCompleted)
+        .def("AssignConditionVariableValuesToNodes", &RansVariableUtilities::AssignConditionVariableValuesToNodes<double>, py::arg("model_part"), py::arg("variable"), py::arg("flag"), py::arg("flag_value") = true)
```

Comment:
Why `model_part`, `variable` and `flag` as kwarg' s? Shouldn't they be always provided?

Reply:
Yes, that's the reason :/